docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md

```diff
@@ -67,8 +67,8 @@ To get the full list of conversion parameters, run the following command:
        .. code-block:: py
           :force:
 
-          from openvino.tools.mo import convert_model
-          ov_model = convert_model(help=True)
+          import openvino as ov
+          ov_model = ov.convert_model(help=True)
```

Comment:
please revert this change.
Most of openvino.tools.**mo**.convert_model() examples won't work for openvino.convert_model().